pkgs/servers/dns/knot-resolver/default.nix

```diff
@@ -26,6 +27,16 @@ unwrapped = stdenv.mkDerivation rec {
 
   outputs = [ "out" "dev" ];
 
+  patches = [
+    (fetchpatch {
+      name = "fix-config-tests-on-darwin.patch";
+      url = "https://gitlab.nic.cz/knot/knot-resolver/-/commit/48ad9d436cf80f58c107774c313a561d852148a0.diff";
+      sha256 = "CEX1XkeYLUSe31xUhNdMRMl1VUXtKFCs5noNJaqL5x0=";
+    })
+  ] ++ lib.optionals (stdenv.isDarwin && stdenv.isAarch64) [
```

Comment:
If we keep this, please make it unconditional

Reply:
Yes, that was the plan, after these OfBorg experiments end.

Reply:
Both `fetchpatch`-ed unconditionally now.